src/root/components/Table/predefinedColumns.ts

```diff
@@ -51,7 +51,8 @@ export function createStringColumn<D, K>(
       value: accessor(datum),
     }),
     valueSelector: accessor,
-    valueComparator: (foo: D, bar: D) => compareString(accessor(foo), accessor(bar)),
+    valueComparator: () => 0,
+    // valueComparator: (foo: D, bar: D) => compareString(accessor(foo), accessor(bar)),
```

Comment:
do we need this?

Reply:
I have not done any changes to this component